.tekton/scripts/check-task-pipeline-bundle-repos.sh

```diff
@@ -27,48 +27,50 @@ locate_in_all_namespaces() {
 
     for quay_namespace in "${CATALOG_NAMESPACES[@]}"; do
         found=$(locate_bundle_repo "$quay_namespace" "$type" "$object")
+        echo "Checking ${quay_namespace}/${type}-${object}, http code: ${found}"
         if [ "$found" != "200" ]; then
-            echo "Missing $type bundle repo: ${quay_namespace}/${type}-${object}"
+            echo "Missing $type bundle repo: ${quay_namespace}/${type}-${object}, creating..."
+            echo "DEBUG: Creating quay repository https://quay.io/${quay_namespace}/${type}-${object}..."
+#            curl --oauth2-bearer "${QUAY_TOKEN}" 'https://quay.io/api/v1/repository' --json '{
+#              "namespace": '"${quay_namespace}"',
+#              "repository": "tekton-catalog/'"${type}-${object}"'",
```

Comment:
The `quay_namespace` already contains `tekton-catalog`. But I don't think that's actually a valid quay API request, the namespace should be just `konflux-ci`.

If it's helpful, locally I have a script that handles namespaces like that automatically

```bash
# konflux-ci/tekton-catalog
if [[ $quay_namespace = */* ]]; then
    # konflux-ci
    quay_namespace=${quay_namespace%%/*}
    # tekton-catalog/
    quay_repo_prefix=${quay_namespace#*/}/
else
    quay_repo_prefix=""
fi

curl --oauth2-bearer "$QUAY_TOKEN" 'https://quay.io/api/v1/repository' --json '{
  "repository": "'"$quay_repo_prefix$repo_name"'",
  "visibility": "public",
  "namespace": "'"$quay_namespace"'",
  "description": ""
}'
```

Reply:
```bash
if [[ $quay_namespace = */* ]]; then 
  # konflux-ci 
  quay_namespace=${quay_namespace%%/*} 
  # tekton-catalog/ 
  quay_repo_prefix=${quay_namespace#*/}/
```
Why does it work for you? Aren't you overwriting the `quay_namespace` to `konflux-ci` before the `quay_repo_prefix`, losing the `tekton-catalog` part in the process? 
Shouldn't the `quay_repo_prefix` in this example be `konflux-ci` because `quay_namespace` is already just stripped `konflux-ci` by that time?
Thanks though, it was helpful